src/main/java/org/cascadebot/cascadebot/music/CascadePlayer.java

```diff
@@ -134,19 +140,19 @@ default void stop() {
         stopTrack();
     }
 
-    default void loadLink(String input, long requestUser, Consumer<String> noMatchConsumer, Consumer<FriendlyException> exceptionConsumer, Consumer<List<AudioTrack>> resultTracks) {
+    default void loadLink(String input, long requestUser, Consumer<String> noMatchConsumer, Consumer<FriendlyException> exceptionConsumer, Consumer<List<AudioTrack>> resultTracks, long channel, long guild) {
```

Comment:
Since this parameter list is getting quite long, it might be better to take requestUser, channel and guild and instead just pass in the TrackData.